src/Iot.Device.Bindings/MathExtension.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+namespace System
+{
+    public static class MathExtension
```

Comment:
Definitely don't love the fact of copying code from corefx. How many bindings depend on this? Would it be feasable to just change the implementation of those bindings (assuming there aren't many of them)?

Reply:
There are 21 usages spread across 6 bindings (12 classes).
I took another stab at it, renaming MathExtension to MathHelper and limiting the #ifs to inside MathHelper. This significantly reduces the amount of #ifs to one. Bindings would have to use MathHelper rather than Math (with the using alias). That way I could also remove the forwarders for Math.Round and Abs.